sample-components-tictactoe/src/app.js

```diff
@@ -62,22 +63,20 @@ export class Game extends React.Component {
     });
   }
 
-  jumpTo(step) {
+  jumpTo (step) {
     this.setState({
       stepNumber: step,
       xIsNext: (step % 2) === 0
     });
   }
 
-  render() {
+  render () {
     const history = this.state.history;
     const current = history[this.state.stepNumber];
     const winner = calculateWinner(current.squares);
 
     const moves = history.map((step, move) => {
-      const desc = move ?
-        'Go to move #' + move :
-        'Go to game start';
+      const desc = move ? ('Go to move #' + move) : 'Go to game start';
```

Comment:
I prefer the old style you had here for readability

Reply:
But I'm not going to block the patch for that

Reply:
Reverting but with ? and : on next line (to make linter happy)